Lib/test/test_pyclbr.py

```diff
@@ -150,20 +150,17 @@ def test_easy(self):
         self.checkModule('difflib', ignore=("Match",))
 
     def test_decorators(self):
-        # XXX: See comment in pyclbr_input.py for a test that would fail
-        #      if it were not commented out.
-        #
         self.checkModule('test.pyclbr_input', ignore=['om'])
 
     def test_nested(self):
         mb = pyclbr
         # Set arguments for descriptor creation and _creat_tree call.
         m, p, f, t, i = 'test', '', 'test.py', {}, None
         source = dedent("""\
-        def f0:
```

Comment:
Was this code running before? This will raise `SyntaxError` because is invalid, no? It is running now?

Reply:
This is something I couldn't figure out. This test was working because of tokenization, and started failing under AST. So I fixed the invalid syntax but I'm not sure if this is a breaking change for this module, I know people may depend on this but this is a low used module and offical documentation says this should be only work on python source codes (which `def f0:` isn't a python code).

Reply:
I think this is fine because I don't think it was a part of the public contract to parse invalid python code in the first place.

Reply:
Me too, I'm resolving this comment.